examples/deploy-aws-lambda/serverless.yml

```
@@ -1,4 +1,4 @@
-service: serverless-bolt-js
+service: serverless-lambda-receiver-bolt-js
```

Update the receiver name to be consistent through the doc:
```suggestion
service: serverless-aws-lambda-receiver-bolt-js
```